app/code/Magento/Security/Model/Config.php

```diff
@@ -27,7 +27,7 @@ class Config implements ConfigInterface
     /**
      * Configuration path to fronted area
      */
-    const XML_PATH_FRONTED_AREA = 'customer/password/';
+    const XML_PATH_FRONTEND_AREA = 'customer/password/';
```

Comment:
Unfortunately we cannot rename constants even if it contained a spelling mistake before :( http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/index.html#removing-or-renaming-constants

Reply:
You could always deprecate the constant with the spelling mistake, create a new constant and use that. Then update the old constant to use the new one, I think that would work.